code/modules/fishing/fish/chasm_detritus.dm

```diff
@@ -94,14 +97,23 @@
 
 	return chasm_contents
 
+/// Body detritus is selected in favor of bodies belonging to sentient mobs
+/// The first sentient body found in the list of contents is returned, otherwise 
+/// if none are sentient choose randomly.
+/obj/item/chasm_detritus/restricted/bodies/determine_detritus(list/chasm_stuff)
+	for(var/thing in chasm_stuff)	
+		var/mob/fallen_mob = thing
```

Comment:
```suggestion
	for(var/mob/fallen_mob in chasm_stuff)
```

important to do it like this for the built-in typecheck or else we'll get runtimes for accessing a non-existent var (`obj` doesn't have `mind`)

Reply:
ah if that's the case can you do it like `for(var/mob/fallen_mob as anything in chasm_stuff)`? is faster since we don't need the typecheck in that case, while reaping the benefits of having the typecast in the `for()`